lib/internal/Magento/Framework/Logger/Formatter/JsonFormatter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\Logger\Formatter;
+
+use Monolog\Formatter\FormatterInterface;
+
+/**
+ * Class JsonFormatter
+ */
+class JsonFormatter extends \Monolog\Formatter\JsonFormatter implements FormatterInterface
+{
+
+    /**
+     * @param array $record
+     * @return array|mixed|string
+     */
+    public function format(array $record)
+    {
+        if (isset($record['context']['exception']) && $record['context']['exception'] instanceof \Exception) {
```

Comment:
Btw, we should take into account `\Throwable` as well now. Or we'll miss errors.
I see that Monolog's JsonFormatter takes this into account when checking the `$data`.

```
if ($data instanceof Exception || $data instanceof Throwable) {
    return $this->normalizeException($data);
}
```